android/java/res_template/xml/file_paths.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) 2020 The Brave Authors. All rights reserved.
+     This Source Code Form is subject to the terms of the Mozilla Public
+     License, v. 2.0. If a copy of the MPL was not distributed with this file,
+     You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<!-- The attributes in this XML file provide configuration information -->
+<!-- for the ContentProvider. -->
+
+<paths xmlns:android="http://schemas.android.com/apk/res/android">
+    <files-path name="images" path="images/"/>
+    <cache-path name="cache" path="net-export/"/>
+    <cache-path name="passwords" path="passwords/"/>
+    <cache-path name="traces" path="traces/"/>
+    <cache-path name="webapk" path="webapks/" />
+    <cache-path name="offline-cache" path="Offline Pages/archives/" />
+
+    <!-- Note(twellington): This path is included to facilitate creating
+         content:// URIs for downloads. This may no longer work if/when users
+         are allowed to save files elsewhere. -->
+    <external-path name="downloads" path="Download/" />
+
+    <!-- Needed for Android Q+, due to scoped storage. -->
```

Comment:
This file is a direct copy from Chromium with the addition of this Android Q+ section. I thought this was simpler than trying to patch for a file of this size, but open to other ideas. Chrome release doesn't seem to (yet) have this problem, because it appears to be built against SDK version 28 still versus 29. I think perhaps this is just due to timing of the release, but it may also warrant further investigation. From what I've seen in the current code base that we build against, however, 29 is now the "default" SDK version.